src/config/dynamic-theme-fixes.config

```diff
@@ -3274,6 +3274,18 @@ CSS
 
 ================================
 
+feedly.com
+
+CSS
+.entry.u0:hover {
```

Comment:
It seems to be that this `hover` over an entry is provided by feedly and thus adding this fix would cause extra functionality than feedly has which isn't the purpose of this `dynamic-theme-fixes.config` file.

Reply:
On the normal white mode you can see the background changing at mouse-over. Also, on the built-in dark mode that Feedly offers you see the background changing at mouse-over. Only when using Dark Reader, the background change is not visible at mouse-over. Hope that makes sense. This is not an extra functionality and hence needs to be fixed.

Reply:
You mean that it has a different colour then `--selected`? Yeah I forgot that that was CSS I injected myself. I changed it to the same colour now.